frontend/src/components/CompareAssignmentsTab/CompareAssignmentsTab.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Flex } from '@instructure/ui-flex/lib/Flex';
+
+import Selector from '../Selector/Selector';
+import AssignmentCard from '../VisCards/AssignmentCard';
+
+const CompareAssignmentsTab = () => (
+  <div>
+    <Flex justifyItems="space-between" margin="0 0 medium">
+      <Flex.Item>
+        <Selector options={['Rubic Name', 'that', 'the other']} labelText="Show Rubric:" />
+      </Flex.Item>
+      <Flex.Item>
+        <Selector options={['Due Date', 'another', 'and another']} labelText="Sort:" />
+      </Flex.Item>
+    </Flex>
+    <div className="filter-bar">
+      <Flex>
+        <Flex.Item>
+          <Selector
+            options={['All Sections', 'section1', 'the other']}
+            labelText="Sections:"
+          />
+        </Flex.Item>
+        <Flex.Item>
+          <Selector
+            options={['All Instuctors', 'Bill', 'Joan']}
+            labelText="Instructors:"
+          />
+        </Flex.Item>
+      </Flex>
+    </div>
+    <AssignmentCard
+      assignmentName="Roanoke Colony Writeup"
+      dueDate={2}
```

Comment:
Just curious, what does the number mean here?

Reply:
It's just a garbage placeholder value to make the mockup work for now. The `2` will render out to `Due 2 days ago`, but in the end we should be passing a date or something.